notes/login/views.py

```diff
@@ -2,20 +2,21 @@
 from .models import User 
 from django.http import JsonResponse
 from django.db import IntegrityError
-
+from .cryptogtapher import Cryptographer
```

Comment:
here's the typo in module name
```suggestion
from .cryptographer import Cryptographer
```